app/src/main/java/com/tyron/code/ui/iconmanager/loader/RadialProgressView.java

```diff
@@ -0,0 +1,217 @@
+package com.tyron.code.ui.iconmanager.loader;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.graphics.drawable.Drawable;
+import android.os.Bundle;
+import android.view.View;
+import android.view.animation.AccelerateInterpolator;
+import android.view.animation.DecelerateInterpolator;
+import androidx.annotation.Keep;
+
+public class RadialProgressView extends View {
```

Comment:
This class is unecessary and does not support XML themes. Use a `ProgressBar` instead.